prime-dotnet-webapi/Services/OrganizationService.cs

```diff
@@ -289,5 +357,12 @@ private IQueryable<Organization> GetBaseOrganizationQuery()
                     .ThenInclude(sa => sa.Addresses)
                         .ThenInclude(pa => pa.Address);
         }
+
+        public async Task<bool> SwitchSigningAuthorityAsync(int organizationId, int newSigningAuthorityId)
+        {
+            var organization = await GetOrganizationAsync(organizationId);
```

Comment:
We don't need the entire `Organization` here with all its related objects, you can just do 
```C#
var organization = await _context.Organizations
    .SingleAsync(o => o.Id == organizationId);
```